ci/evalml_integration_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -x
+
+# https://stackoverflow.com/questions/42218009/how-to-tell-if-any-command-in-bash-script-failed-non-zero-exit-status/42219754#42219754
+# Set err to 1 if pytest failed.
+error=0
+trap 'error=1' ERR
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+cd "$GIT_ROOT" || exit
+
+# Install EvalML
+# TODO uncomment once evalml 0.24.0 is released, which adds support for pickle
+#pip install evalml>=0.24.0
+pip install git+https://github.com/alteryx/evalml.git@main
```

Comment:
TODO delete this change before getting reviews. This a testing hack until evalml v0.24.0 goes out, which adds support for native python `pickle` serialization.